starter/source/tools/curve/hm_read_funct_python.F90

```diff
@@ -158,17 +171,27 @@ subroutine hm_read_funct_python(python,npc,snpc,total_nb_funct,&
                 allocate(table(l)%Y)
                 allocate(table(l)%X(1)%values(funct_python_nsamples))
                 allocate(table(l)%Y%values(funct_python_nsamples))
-                call python_sample_function(python%functs(i)%name,XX,YY,funct_python_nsamples)
+                write(6,*) "Python test: funct_id",func_id,"name is",array_to_string(python%functs(i)%name)
```

Comment:
@laurent-altr 

I think you forget to remove or comment some printing.

Sébastien